actionpack/lib/abstract_controller/callbacks.rb

```diff
@@ -81,83 +81,107 @@ def _insert_callbacks(callbacks, block = nil)
       # :call-seq: before_action(names, block)
       #
       # Append a callback before actions. See _insert_callbacks for parameter details.
+      # Calling the same callback multiple times in +before_action+ will overwrite
```

Comment:
Not sure if this is the right place or way for adding this docs on controller callbacks ?